src/microbe_stage/MicrobeHUD.cs

```diff
@@ -243,6 +247,9 @@ public class MicrobeHUD : Node
     /// </summary>
     private float hoverInfoTimeElapsed;
 
+    private System.Collections.Generic.Dictionary<string, ActionButton> abilitiesHotkey =
```

Comment:
Do these *need* to be stored in a dictionary? If the names are already known and there really isn't any operation that directly runs on all of them, wouldn't it be better to put them all to separate variables so that dictionary lookups based on strings don't have to happen each frame?

Reply:
I guess yeah the buttons themselves are not very dynamic, I'll change this to use the separate variables approach.